static/.eslint-ce-static.yml

```diff
@@ -0,0 +1,85 @@
+---
+plugins:
+  - jsdoc
+  - sonarjs
+  - unicorn
+extends:
+  - ../.eslint-license-header.yml
+  - eslint:recommended
+env:
+  browser: true
+  node: true
+  es6: false
```

Comment:
I suppose we could add a comment here to enable es6 once all the old es5 js code is gone?

Reply:
As long as webpack does magic to keep it es5, a couple more browsers will otherwise break.

Reply:
Agreed: or at least the option (I suspect ES5 is already broken by Monaco et al...)